manifests/plugin/battery.pp

```diff
@@ -0,0 +1,17 @@
+# https://collectd.org/wiki/index.php/Plugin:Uptime
+class collectd::plugin::battery (
+  $ensure   = 'present',
+  $interval = undef,
+  Optional[Boolean] $values_percentage = undef,
```

Comment:
can we instead default to true or false and don't mark it as optional?

Reply:
done